src/test/kotlin/graphics/scenery/tests/examples/volumes/TransferFunctionEditorExample.kt

```diff
@@ -0,0 +1,144 @@
+package graphics.scenery.tests.examples.volumes
+
+import bdv.spimdata.XmlIoSpimDataMinimal
+import graphics.scenery.*
+import graphics.scenery.backends.Renderer
+import graphics.scenery.controls.SwingBridgeFrame
+import graphics.scenery.controls.SwingUiNode
+import graphics.scenery.utils.extensions.plus
+import graphics.scenery.utils.extensions.times
+import graphics.scenery.volumes.Colormap
+import graphics.scenery.volumes.TransferFunctionUI
+import graphics.scenery.volumes.Volume
+import org.joml.Vector3f
+import org.scijava.ui.behaviour.ClickBehaviour
+import org.scijava.ui.behaviour.DragBehaviour
+import tpietzsch.example2.VolumeViewerOptions
+import java.io.File
+
+
+/**
+ * @author Konrad Michel
+ * Example for scenery - swing bridge
+ *
+ * A TransferFunctionEditor example to add, manipulate and remove control points of a volume's transfer function.
+ * Further more able to generate a histogram representation of the volume data distribution to help with the transfer function setup.
+ */
+class TransferFunctionEditorExample : SceneryBase("TransferFunctionEditor Example", 1280, 720, false) {
+    var maxCacheSize = 512
+    val cam: Camera = DetachedHeadCamera()
+
+    override fun init() {
```

Comment:
Kdoc missing

Reply:
SHould I doc the init function? The example itself has an emplanation to it, or what am I missing here?